### PR TITLE
Adding create-topic script.

### LINKED ISSUE #637: Setup and test update the bridge config from HCS topics messages
@svetlio8 commented on [Tue Jun 07 2022](https://github.com/LimeChain/hashport-api/issues/75)

Since we already have all validators with the image that includes the Update the bridge config from HCS topics messages #601 feature, we can test by creating the topic and updating the configuration on testnet.

Add script for topic creation with admin and submit keys.

### PR DESCRIPTION
**Pull request by Psykepro**
Signed-off-by: Nikolay Nedkov <nikolay.nedkov@limechain.tech>

**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #637

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: scripts/common/create-topic/create-topic.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	}
+	var memberKeys []hedera.PrivateKey
+	memberKeys = append(memberKeys, pK)
+
+	fmt.Println("Private keys array:", memberKeys)
+
+	topicKey := hedera.KeyListWithThreshold(uint(*members))
+	for i := 0; i < *members; i++ {
+		topicKey.Add(memberKeys[i].PublicKey())
+	}
```
Comment (failfmi) [inline scripts/common/create-topic/create-topic.go:55]: If you set members to > 1, this will always panic, as you are trying to access index out of bound

**Review by failfmi — CHANGES_REQUESTED** [review]

File: scripts/common/create-topic/create-topic.go (hunk 1 above)
Comment (Psykepro) [reply scripts/common/create-topic/create-topic.go:55]: ops yep

**Review by svetlio8 — DISMISSED** [review]

File: scripts/README.md [hunk 2]
```diff
@@ -14,11 +14,14 @@
    `go run ./scripts/token/associate/associate.go --privateKey=/your private key/ --accountID=/your account id/ --network=/previewnet|testnet|mainnet/ --tokenID=/The Token id from the output of the previous step/`
 
 ## Update bridge config
-1. Run prepare-update-config.go with executorAccountID, topicID and configPath to create frozen transaction as bytes which to send for signing to all members
+1. Run create-topic.go (only once) with supplyKeys, executorAccountID, executorPrivateKey, keyThreshold, network to create the topic with all members public keys and the key threshold.
```
Comment (failfmi) [inline scripts/README.md:17]: 1. Run create-topic.go (only once) with supplyKeys, executorAccountID, executorPrivateKey, keyThreshold, network to create the topic with all given public keys and the key threshold.
   `go run ./scripts/common/create-topic/create-topic.go --supplyKeys /public keys, split by comma/ --executorAccountID /executor account id/ --executorAccountID /executor private key/ --keyThreshold /key threshold (default: 1)/ --network /previewnet|testnet|mainnet/`

File: scripts/README.md [hunk 3]
```diff
@@ -14,11 +14,14 @@
...
+   `go run ./scripts/common/create-topic/create-topic.go --supplyKeys /members public keys/ --executorAccountID /executor account id/ --executorAccountID /executor private key/ --keyThreshold /key threshold (default: 1)/ --network /previewnet|testnet|mainnet/`
+
+2. Run prepare-update-config.go with executorAccountID, topicID and configPath to create frozen transaction as bytes which to send for signing to all members
    `go run ./scripts/bridge/update-config/prepare-update-config.go --executorAccountID /executor account id/ --topicID /topic id/ --network /previewnet|testnet|mainnet/ --configPath /full or relative path to bridge.yml/ --validStartMinutes /start minutes (default 2)/ --nodeAccountID /node account id which process the transaction (default: "0.0.3")/`
 
-2. Every member of the topic needs to run sign on the generated bytes by running sign-transaction.go with privateKeys and transaction (the bytes)
+3. Every member of the topic needs to run sign on the generated bytes by running sign-transaction.go with privateKeys and transaction (the bytes)
```
Comment (failfmi) [inline scripts/README.md:23]: Depending on the threshold, n out of m keys of the threshold supply key have to sign the generated bytes by running `sign-transaction.go` with privateKeys and transaction itself

File: scripts/README.md [hunk 4]
```diff
@@ -14,11 +14,14 @@
...
    `go run ./scripts/common/sign/sign-transaction.go --privateKeys /member private key (or all members private keys separated by comma/ --transaction /transaction bytes/`
```
Comment (failfmi) [inline scripts/README.md:24]: `go run ./scripts/common/sign/sign-transaction.go --privateKeys /private keys, which need to sign the transaction, separated by comma/ --transaction /transaction bytes/`

File: scripts/README.md [hunk 5]
```diff
@@ -14,11 +14,14 @@
...
 
-3. Submit the signed transactions by all members by running submit-transaction.go with privateKey, accountID, network and transaction (the signed bytes)
+4. Submit the signed transactions by all members by running submit-transaction.go with privateKey, accountID, network and transaction (the signed bytes)
```
Comment (failfmi) [inline scripts/README.md:26]: Submit the transaction after it has been signed by the required keys, by running submit-transaction.go with privateKey, accountID, network and transaction (the signed bytes)

**Review by failfmi — APPROVED** [review]

**Pull request merged**